tutorials/main.js

```diff
@@ -0,0 +1,2 @@
+/*! For license information please see main.js.LICENSE.txt */
```

Comment:
We should avoid committing compiled code where possible, ideally by [running webpack from node](https://masteringjs.io/tutorials/webpack/node) when running `node .`.

Also, disable minification next time: https://davidwalsh.name/how-to-not-minify-source-with-webpack .